src/main/java/seedu/address/logic/commands/EditCommand.java

```diff
@@ -120,6 +121,14 @@ public boolean equals(Object other) {
                 && editPersonDescriptor.equals(e.editPersonDescriptor);
     }
 
+    @Override
+    public String toString() {
+        return new ToStringBuilder(this)
+                .add("index", index)
```

Comment:
Hi, I think having a uniform toString() format is a great idea!
However, I am not sure if it is expected behaviour for this index to be printed as an index object hash code (eg. seedu.address.commons.core.index.Index@22d9bc14). 
Perhaps the index class requires a toString() method as well if this is not intended.

Reply:
Good catch @cjyothika 👍